webpack.config.electron.js

```diff
@@ -1,12 +1,18 @@
 /* eslint-env node */
 const webpack = require('webpack');
 const HappyPack = require('happypack');
+const path = require('path');
 
 module.exports = env => {
   const entry = env && env.LINUX ? './server/main.dev.linux.js' : './server/main.dev.js';
 
   return {
     entry: entry,
+    resolve: {
+      alias: {
+        jsbi: path.resolve(__dirname, 'node_modules', 'jsbi', 'dist', 'jsbi-cjs.js')
```

Comment:
what's this?

Reply:
I'm not sure I need to look into why this works. @martpie came up with this. See [this issue](https://github.com/emersion/mpris-service/issues/23) on mpris-service.

Reply:
JSBI ships with a `.mjs` version of the library. The thing is Webpack's `electron-main` will try to import this `.mjs` instead of the classic common JS file. Then things goes terribly wrong.

To me, this solution is a hack, but I am not sure if the problem comes from `jsbi` builds, or `node-dbus-next` (which seems to be written in cjs, but without a transpiler/bundler)